libc-test/build.rs

```diff
@@ -328,6 +328,9 @@ fn test_apple(target: &str) {
 
             // FIXME: XCode 13.1 doesn't have it.
             "TIOCREMOTE" => true,
+
+            // Private value used by debuggers.
+            "_POSIX_SPAWN_DISABLE_ASLR" => true,
```

Comment:
I don't think we should expose private consts on libc, it's typically unstable and libc versioning doesn't fit it.